docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -208,3 +208,9 @@ on the number of agents required by your deployment:
 | 100,000          | 8GB x2                | 8 vCPU x2           | 256GB x2 RAM \| 64 vCPU x2
 |===
 
+[discrete]
+[[agent-policy-scale]]
```

Comment:
```suggestion
[[agent-policy-scaling-recommendations]]
```
This should fix the failing Docs CI check (caused by a duplicate ID from `agent-policies.asciidoc`).